ash/src/vk.rs

```diff
@@ -19,19 +19,19 @@ macro_rules! vk_make_version {
 #[macro_export]
 macro_rules! vk_version_major {
     ($major:expr) => {
-        ($major as uint32_t) >> 22
+        ($major as u32) >> 22
```

Comment:
I think we should use something like `crate::ash::vk::uint32_t`, but I don't think that quite works if the crate renames ash. I think there was a correct way but I forgot how to do it as I don't write macros that often.

Reply:
I think will work `$crate::vk::uint32_t`. But before I make this change, any reason why we don't use Rust's own `u32`? It's statically guaranteed to match Vulkan's 32-bit C unsigned integers.

Reply:
I mean it doesn't really matter, but ash uses libc everywhere else atm, so this would be the most consistent. Maybe we should switch away from libc and use rust std lib. I am honestly not sure.